web/themes/custom/cloudy/templates/content/events/node--event.html.twig

```diff
@@ -10,34 +10,13 @@
       {{ content.field_featured_media }}
       <div class="mt-8 mb-8">
         {{ content.group_date }}
-        {% if content.field_all_day.0|render|trim is not empty or content.field_start_time.0 is not empty or content.field_end_time.0 is not empty %}
-          <div class="event_time d-flex">
-            {% if content.field_all_day.0|render|trim is not empty %}
-                {{ content.field_all_day }}
-              {% elseif content.field_start_time.0 is not empty and content.field_end_time.0 is not empty %}
-                {{ content.field_start_time }}
-                <span class="mx-1" aria-label="{{ 'to'|t }}">&ndash;</span>
-                {{ content.field_end_time }}
-              {% elseif content.field_start_time.0 is empty and content.field_end_time.0 is not empty %}
-                <span class="me-1">{{ 'ends'|t|capitalize }} </span>
-                {{ content.field_end_time }}
-              {% else %}
-                {{ content.field_start_time }}
-              {% endif %}
-          </div>
-        {% endif %}
       </div>
-      {# Hide link from CSM-15 until form is ready to publish.
-      <div class="event-ada-link d-flex">
-        <a href="/oehr/request-ada-accommodation?request_what=event_accommodations&name={{ label|render|striptags|trim|url_encode }}&date={{ content.field_start_date.0|render|striptags|trim|url_encode }}&location={{ content.field_location.0['#node'].title.value|render|striptags|trim|url_encode }}&host={{ content.field_display_groups.0['#title']|render|striptags|trim|url_encode }}&url={{ url('<current>')|render|url_encode }}&autocomplete=yes">Request ADA Accommodation</a>
-      </div> #}
       <div class="mb-8">
         {{ content.field_is_online }}
-        {% if content.field_is_online.0|render|trim %}
+        {% if node.field_is_online.value %}
```

Comment:
For my own education: what's the reason to switch from content to node here?

Reply:
Good question, I had to refresh my memory on this one as well. I wanted to be able to check the value without having to render it. That field is set up with a custom boolean formatter:

![image](https://github.com/user-attachments/assets/ca6f2155-0c0a-4194-9844-61a208548eaf)

So the value of that field in the content array is a markup string:
Example when is_online == true:

![image](https://github.com/user-attachments/assets/65dc3f72-c577-404b-b1d7-eebc11eace29)

Example when is_online == false:

![image](https://github.com/user-attachments/assets/e3115907-7a48-4fbc-915c-9416419fff80)

While we could technically check `content.field_is_online['#items'].0.value` or `content.field_is_online.0['#markup'] != ''`, I figured semantically, since we're using it in an `if` statement, it makes more sense to use the `node` array since that will contain the actual boolean value from the entity.

Reply:
Thanks for the explanation! Will be very useful when I encounter similar situation in the future.